src/crosshook-native/src/styles/theme.css

```
@@ -934,6 +945,358 @@ textarea {
+  color: var(--crosshook-color-text);
+  line-height: 1.55;
+}
+
+.crosshook-protondb-card__banner-copy--muted {
+  color: var(--crosshook-color-text-muted);
+}
+
+.crosshook-protondb-card__source-link {
+  color: var(--crosshook-color-accent-strong);
+  font-weight: 600;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)